pkgs/development/tools/poac/default.nix

```
@@ -0,0 +1,43 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, git
+, cacert
+, boost179
+, fmt_8
+, libarchive
+, libgit2
+, llvmPackages_14
+, lz4
+, openssl
+, spdlog
+}:
+
+stdenv.mkDerivation rec {
+  pname = "poac";
+  version = "0.3.7";
+
+  src = fetchFromGitHub {
+    owner = "poacpm";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-mrEtLyVnv6jggQICie210TZiG20eAacbjVXFlZqleVE=";
+  };
+
+  cmakeFlags = [
+    "-DCPM_USE_LOCAL_PACKAGES=ON"
+    "-DBOOST_ASIO_DISABLE_STD_ALIGNED_ALLOC=1" # https://github.com/msys2/MINGW-packages/issues/9827#issuecomment-947985996
+    "-DBoost_NO_BOOST_CMAKE=ON" # https://github.com/msys2/MINGW-packages/issues/5233#issuecomment-485244682
+  ];
+
+  nativeBuildInputs = [ cmake git cacert llvmPackages_14.clang ];
+  buildInputs = [ boost179 fmt_8 libarchive libgit2 lz4 openssl spdlog ];
+
+  meta = with lib; {
+    description = "Package manager for C++";
+    homepage = "https://github.com/poacpm/poac";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ ken-matsui ];
```

```suggestion
    homepage = "https://poac.pm/";
    description = "Package manager for C++";
    license = licenses.asl20;
    maintainers = with maintainers; [ ken-matsui ];
    platforms = platforms.unix;
```

If there is another homepage besides github, use it.

Optionally you can add a `longDescription`.